src/Components/CardboardList/CardboardListItem.tsx

```diff
@@ -42,7 +43,6 @@ export const CardboardListItem = <T extends unknown>(
     const showWarningIndicator = isValid === false;
 
     // end icon
-    const isEndIconCustomRender = typeof iconEnd !== 'object';
```

Comment:
Why was this removed?

Reply:
cause it turns out I only used it in that useEffect and it wasn't actually used in there anymore. Woops.